src/main/java/core/Cars.java

```diff
@@ -0,0 +1,50 @@
+package core;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Cars {
+    private static int WINNER_POSITION;
+    private List<Car> cars;
+
+    public Cars(String[] carNames) {
+        cars = new ArrayList<>();
+        Arrays.stream(carNames).forEach(carName -> cars.add(Car.create(carName)));
+    }
```

Comment:
```suggestion
    private Cars(List<Car> cars) {
        this.cars = cars;
    }
    
    public static of(String[] carNames) {
        return Arrays.stream(carNames)
                    .mapToObj(Car::create)
                    .collect(collectingAndThen(toList(), Cars::new));
    }
```
이런식으로 표현할 수 있습니다!
생성자는 생성자의 역할만 수행하도록 만드는거죠